apps/src/StudioApp.js

```diff
@@ -1185,7 +1185,7 @@ StudioApp.prototype.setConfigValues_ = function (config) {
 
 // Overwritten by applab.
 StudioApp.prototype.runButtonClickWrapper = function (callback) {
-  $(window).trigger('run_button_pressed');
+  $(window).trigger('appModeChanged');
```

Comment:
Will change this so that the run button fires run_button_pressed and appModeChanged, and other toggles fire just appModeChanged